src/main/resources/static/bloodDrives.html

```diff
@@ -0,0 +1,283 @@
+        success: function(){
+            location.reload();
+        },
+        error: function(){
+            alert("Error Failed To Delete all outdated blood drives");
+        }
+    });
+}
+
+
+  function removeChildNodes(parent){
+    while (parent.firstChild) {
+        parent.removeChild(parent.lastChild);
+    }
+  }
+
+function showMessages(data){
+    let bloodDriveList = document.querySelector("#feedbackList");
+    // clear all messages
+    removeChildNodes(feedbackList);
+    let messageTemplate = document.getElementById("commentTemplate");
+
+    // loop through each message and display it.
+
+    for(var i=0; i < data.length; i++){
+        // create a new message element from the template
+        let newMessageElement = messageTemplate.content.cloneNode(true);
+
+        // get the text component of the message
+        let textComponent = newMessageElement.querySelector('.text');
+        // set the message to the current message
+        let format = "<hr/>" + data[i].title + "<br/>"
+          + "Locaiton: " + data[i].location + "<br/>"
+          + "Date: <br/>" + data[i].date + "<br/>"
+          + "Time: " + data[i].time + "<br/>"
+          + "Link: " + data[i].link + "<br/>";
+
+          textComponent.innerHTML = format;
+
+          bloodDriveList.appendChild(newMessageElement);
+
+
+      }
+
+  }
+
+function showAdminMessages(data){
+    let bloodDriveList = document.querySelector("#feedbackList");
+    // clear all messages
+    removeChildNodes(feedbackList);
+    let messageTemplate = document.getElementById("commentTemplate");
+
+    // loop through each message and display it.
+
+    for(var i=0; i < data.length; i++){
+        // create a new message element from the template
+        let newMessageElement = messageTemplate.content.cloneNode(true);
+
+        // get the text component of the message
+        let textComponent = newMessageElement.querySelector('.text');
+        // set the message to the current message
+        let format = "<hr/>" + data[i].title + "<br/>"
+          + "Locaiton: " + data[i].location + "<br/>"
+          + "Date: <br/>" + data[i].date + "<br/>"
+          + "Time: " + data[i].time + "<br/>"
+          + "Link: " + data[i].link + "<br/>";
+
+          textComponent.innerHTML = format;
+
+          var button = document.createElement('input');
+
+          // SET INPUT ATTRIBUTE 'type' AND 'value'.
+          button.setAttribute('type', 'button');
+          button.setAttribute('class', 'ui red inverted button');
+          button.setAttribute('value', 'Delete');
+          button.setAttribute('id', data[i].id);
+
+          // ADD THE BUTTON's 'onclick' EVENT.
+          button.setAttribute('onclick', 'deleteButton(this.id)');
+
+          bloodDriveList.appendChild(newMessageElement);
+          bloodDriveList.appendChild(button);
+
+
+      }
+
+  }
+
+  function fetchMessages(){
+    $.ajax({
+        type: 'GET',
+        dataType: 'json',
+        url: '/bloodDrive',
```

Comment:
URL should be /bloodDrive/upcoming 
This way regular users only see the upcoming blood drives and the ones that are outdated are filtered out.